build_projects/dotnet-host-build/PackageTargets.cs

```diff
@@ -49,6 +50,41 @@ public static BuildTargetResult GenerateVersionBadge(BuildTargetContext c)
         }
         
         [Target]
+        [BuildPlatforms(BuildPlatform.Windows)]
+        public static BuildTargetResult CopyNativeSymbols(BuildTargetContext c)
+        {
+            var nativeSymbolsRoot = Path.Combine(Dirs.Output, "obj", "nativeSymbols");
+            var nativeSymbolsZipFileDir = Path.Combine(Dirs.Intermediate, "nativeSymbols");
+
+            Utils.DeleteDirectory(nativeSymbolsRoot);
+            Directory.CreateDirectory(nativeSymbolsRoot);
+
+            Utils.DeleteDirectory(nativeSymbolsZipFileDir);
+            Directory.CreateDirectory(nativeSymbolsZipFileDir);
+
+            foreach (var file in Directory.GetFiles(Dirs.CorehostLatest, "*.pdb", SearchOption.TopDirectoryOnly))
+            {
+                var destFile = file.Replace(Dirs.CorehostLatest, nativeSymbolsRoot);
+                File.Copy(file, destFile, true);
+                c.Warn(destFile);
+            }
+            FixPermissions(nativeSymbolsRoot);
+            
+            var expectedFiles = new string[] { "dotnet.pdb", "hostfxr.pdb", "hostpolicy.pdb" };
```

Comment:
In CoreFX and CoreCLR we strip xplat symbols into `.so.dbg` and `.dylib.dwarf` files. By default, symbols might still exist in the `.so` and `.dylib` files, but I don't know for sure about core-setup.